awx/ui_next/src/screens/Setting/SettingList.jsx

```diff
@@ -25,10 +25,11 @@ const SplitLayout = styled(PageSection)`
   }
 `;
 const Card = styled(_Card)`
-  display: inline-block;
-  break-inside: avoid;
-  margin-bottom: 24px;
-  width: 100%;
+  && {
```

Comment:
Nice. I didn't realize that PatternFly's card styles were overriding our display property. 🔥